stubs/openpyxl/openpyxl/styles/fonts.pyi

```diff
@@ -1,58 +1,95 @@
+    UNDERLINE_SINGLE: Final = "single"
+    UNDERLINE_SINGLE_ACCOUNTING: Final = "singleAccounting"
+    name: str | None
+    @property
+    def charset(self) -> int | None: ...
+    @charset.setter
+    def charset(self, __value: _IntegerSetter | None) -> None: ...
+    @property
+    def family(self) -> float | None: ...
+    @family.setter
+    def family(self, __value: _FloatSetter | None) -> None: ...
+    @property
+    def sz(self) -> float | None: ...
+    @sz.setter
+    def sz(self, __value: _FloatSetter | None) -> None: ...
+    size = sz
+    @property
+    def b(self) -> bool: ...
+    @b.setter
+    def b(self, __value: _BoolSetter) -> None: ...
+    bold = b
+    @property
+    def i(self) -> bool: ...
+    @i.setter
+    def i(self, __value: _BoolSetter) -> None: ...
+    italic = i
+    @property
+    def strike(self) -> bool | None: ...
+    @strike.setter
+    def strike(self, __value: _BoolSetter) -> None: ...
+    strikethrough = strike
+    @property
+    def outline(self) -> bool | None: ...
+    @outline.setter
+    def outline(self, __value: _BoolSetter) -> None: ...
+    @property
+    def shadow(self) -> bool | None: ...
+    @shadow.setter
+    def shadow(self, __value: _BoolSetter) -> None: ...
+    @property
+    def condense(self) -> bool | None: ...
+    @condense.setter
+    def condense(self, __value: _BoolSetter) -> None: ...
+    @property
+    def extend(self) -> bool | None: ...
+    @extend.setter
+    def extend(self, __value: _BoolSetter) -> None: ...
+    u: Literal["single", "double", "singleAccounting", "doubleAccounting", None]
+    underline = u  # noqa: F821
+    vertAlign: Literal["superscript", "subscript", "baseline", None]
+    color: Color | None
+    scheme: Literal["major", "minor", None]
+
     tagname: str
-    __elements__: Incomplete
+
+    __elements__: tuple[str, ...]
     def __init__(
         self,
-        name: Incomplete | None = ...,
-        sz: Incomplete | None = ...,
-        b: Incomplete | None = ...,
-        i: Incomplete | None = ...,
-        charset: Incomplete | None = ...,
-        u: Incomplete | None = ...,
-        strike: Incomplete | None = ...,
-        color: Incomplete | None = ...,
-        scheme: Incomplete | None = ...,
-        family: Incomplete | None = ...,
-        size: Incomplete | None = ...,
-        bold: Incomplete | None = ...,
-        italic: Incomplete | None = ...,
-        strikethrough: Incomplete | None = ...,
-        underline: Incomplete | None = ...,
-        vertAlign: Incomplete | None = ...,
-        outline: Incomplete | None = ...,
-        shadow: Incomplete | None = ...,
-        condense: Incomplete | None = ...,
-        extend: Incomplete | None = ...,
+        name: str | None = ...,
+        sz: _FloatSetter | None = ...,
+        b: _BoolSetter = ...,
+        i: _BoolSetter = ...,
+        charset: _IntegerSetter | None = ...,
+        u: Literal["single", "double", "singleAccounting", "doubleAccounting", None] = ...,
```

Comment:
Looks like this type appears several times in this file; maybe it'd be another good candidate for a type alias